server/routes/serviceProviderReferrals/referralOverviewPagePresenter.test.ts

```diff
@@ -0,0 +1,36 @@
+import serviceUserFactory from '../../../testutils/factories/deliusServiceUser'
+import sentReferralFactory from '../../../testutils/factories/sentReferral'
+
+import ReferralOverviewPagePresenter, { ReferralPageSection } from './referralOverviewPagePresenter'
+
+describe(ReferralOverviewPagePresenter, () => {
+  describe('serviceUserBannerArgs', () => {
+    it('returns the embedded serviceUserBannerArgs', () => {
+      const serviceUser = serviceUserFactory.build()
+      const sentReferral = sentReferralFactory.build()
+      const presenter = new ReferralOverviewPagePresenter(ReferralPageSection.Details, sentReferral, serviceUser)
+      expect(presenter.serviceUserBannerArgs).toBeDefined()
```

Comment:
What is this testing that isn't already sufficiently guaranteed by the type system?

Reply:
i think this is a legitimate test which guarantees that future refactors do not remove `serviceUserBannerArgs`. i think if it was removed calls of the get would just get an undefined value and it would not cause compile time errors

Reply:
It depends on who's calling it - if it's called by TypeScript code then it'll give a compile error I think. Do you mean if it's called from a template?